settings.yml

```diff
@@ -30,6 +30,10 @@ show_below_site_header:
   type: bool
   description: "Display the banner in the below site header connector."
 
+show_above_main_container:
```

Comment:
I believe three options (`show_above_site_header`, `show_below_site_header` and now `show_above_main_container`) are too many control just the position. It would be better to have a single `enum`:

```suggestion
position:
  default: above_site_header
  type: enum
  description: "Display the banner in the above site header, below site header or above main container connector."
  choices:
    - above_site_header
    - below_site_header
    - above_main_container
```